community/lld/APKBUILD

```diff
@@ -45,11 +45,12 @@ check() {
 		armhf|armv7) rm ../test/ELF/eh-frame-hdr-augmentation.s;;
 	esac
 
-	#make check-lld
+	# make check-lld doesn't like using system lit
```

Comment:
I was trying with `LLVM_EXTERNAL_LIT` in #8533

Reply:
I think calling lit manually should do the same thing, but thanks for the pointer!